src/java.desktop/share/classes/java/awt/Cursor.java

```diff
@@ -300,16 +300,15 @@ public static Cursor getSystemCustomCursor(final String name)
             String prefix = CURSOR_DOT_PREFIX + name;
```

Comment:
I'd mark `prefix` as `final`, it's used throughout the method, which highlights the inconsistency in applying `final`.